WordPress/wordcamp.org
# Add Events shortcode

Allows the creation of Events shortcode to gather application data, rather than using PD form.

Testing this is somewhat tricky due to it saving data on Central, and there being no test environment to check.

I've done some limited testing without saving or sending notifications to slack, but will be best to try this after a merge and deploy, with some mock events to make sure it's all working correctly.

Fixes #881

Testing instructions:

Add the shortcode `[events-organizer-application]` to a page (can be draft) - complete the form, and the data should save as a `WordCamps` post type with all data saved. Only some fields are mapped into the post type.
